### PR TITLE
Cypress. Fix for Firefox browser.

### LINKED ISSUE #2792: Nightly build is failed (.cvat-draw-polygon-popover-visible never found)
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

CI tests have to be robust.

### Current Behaviour

```
  2) Actions on polygon
       Testing case "10"
         Draw a polygon shape, track with second label:
     AssertionError: Timed out retrying: Expected to find element: `.cvat-draw-polygon-popover-visible`, but never found it.
      ./cypress/support/commands.js/<@http://localhost:8080/__cypress/tests?p=cypress/support/index.js:1035:6
      runCommand/<@http://localhost:8080/__cypress/runner/cypress_runner.js:169097:15
      tryCatcher@http://localhost:8080/__cypress/runner/cypress_runner.js:10325:23
```

### Steps to Reproduce (for bugs)

https://github.com/openvinotoolkit/cvat/actions/runs/556084657

### Context

Nightly testing on our CI

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Trying to solve the crash of tests in nightly.
Test run in CI: https://github.com/dvkruchinin/cvat/pull/19/checks?check_run_id=1881416466
Resolved #2792 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -146,38 +146,38 @@ Cypress.Commands.add('openTaskJob', (taskName, jobID = 0, removeAnnotations = tr
     cy.openJob(jobID, removeAnnotations);
 });
 
+Cypress.Commands.add('interactControlButton', (objectType) => {
+    cy.get('body').click('top');
+    cy.get(`.cvat-${objectType}-control`).trigger('mouseleave').trigger('mouseout').trigger('mouseover');
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:151]: .click() is not enough?

File: tests/cypress/support/commands.js [hunk 2]
```diff
@@ -146,38 +146,38 @@ Cypress.Commands.add('openTaskJob', (taskName, jobID = 0, removeAnnotations = tr
     cy.openJob(jobID, removeAnnotations);
 });
 
+Cypress.Commands.add('interactControlButton', (objectType) => {
+    cy.get('body').click('top');
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:150]: Is clicking on exactly 'top' important?

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js:151]: cy.click () contains a certain number of mouse events. Perhaps this is one of the reasons for incorrect interaction with the button. I think trigger() as a lower-level command will be more optimal.

File: tests/cypress/support/commands.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js:150]: click() without specifying coordinates produces mouse events in the center of the object. In this case, it turns out that he clicks on the canvas. There may also be a drawn object under the cursor. Which can lead to unexpected consequences.

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (bsekachev) [reply tests/cypress/support/commands.js:151]: > cy.click () contains a certain number of mouse events. 

What do you mean? cy.click just invokes click function of the element. You can do the same using  window.document.body.click() as far as I understand.

File: tests/cypress/support/commands.js (hunk 2 above)
Comment (bsekachev) [reply tests/cypress/support/commands.js:150]: Did you try using focus() instead of clicking?

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js:151]: cy.click() contains events:
```
pointerover
mouseover
pointermove
mousemove
pointerdown
mousedown
pointerup
mouseup
click
```

File: tests/cypress/support/commands.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js:150]: I'll double-check. Maybe it would be better.

Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Changed to focus().

**Review by bsekachev — APPROVED** [review]

**Pull request merged**